changelog/4056.bugfix.rst

```diff
@@ -0,0 +1,4 @@
+Ensure ``MonkeyPatch.setenv`` and ``MonkeyPatch.delenv`` use ``bytes`` for environment variable names in Python 2.
```

Comment:
this description is incorrect, we use "native strings"

Reply:
True, thanks for the pointer. Fixed. 👍